AlphaWallet/Tokens/ViewControllers/TokensViewController.swift

```diff
@@ -270,22 +270,23 @@ extension TokensViewController: UITableViewDelegate {
         let token = viewModel.item(for: indexPath.row, section: indexPath.section)
 
         switch token.type {
-        case .ether:
+        case .ether, .xDai:
             let cellViewModel = EthTokenViewCellViewModel(
                     token: token,
                     ticker: viewModel.ticker(for: token),
                     currencyAmount: session.balanceCoordinator.viewModel.currencyAmount,
-                    currencyAmountWithoutSymbol: session.balanceCoordinator.viewModel.currencyAmountWithoutSymbol
+                    currencyAmountWithoutSymbol: session.balanceCoordinator.viewModel.currencyAmountWithoutSymbol,
+                    chainId: session.config.chainID
```

Comment:
Can we pass in `session.config.server` instead of `session.config.chainID`. Stronger type check, etc.